src/components/routes/CalendarItemDisplay.vue

```diff
@@ -0,0 +1,241 @@
+            />
+          </svg>
+        </q-icon>
+      </template>
+
+      <!-- Content: Route logged -->
+      <template v-if="route && route.id">
+        <!-- Icon: Transport type -->
+        <q-icon
+          :color="active ? 'white' : 'primary'"
+          :name="getRouteIcon(route.transport)"
+          size="18px"
+          data-cy="calendar-item-icon-transport"
+        />
+        <!-- Distance -->
+        <span
+          class="relative-position text-subtitle2"
+          :class="[active ? 'text-white' : 'text-primary']"
+          data-cy="calendar-item-distance"
+        >
+          {{ route.distance }} km
```

Comment:
Unit should be global var?

Reply:
Yes, refactored into a new composable function in `useRoutes.ts`.